bin/cli.js

```diff
@@ -1,15 +1,35 @@
 #!/usr/bin/env node
 
+var ok = require('log-ok');
 var deleteEmpty = require('..');
+var DRY_RUN = '--dry-run';
+
 var argv = process.argv.slice(2);
-var cwd = argv[0] || process.cwd();
+var dryRun = false;
+if (argv[0] === DRY_RUN || argv[1] === DRY_RUN) {
+  dryRun = true;
+}
+
+var cwd = process.cwd();
+if (argv[0] && argv[0] !== DRY_RUN) {
+  cwd = argv[0];
+} else if (argv[1] && argv[1] !== DRY_RUN) {
+  cwd = argv[1];
+}
```

Comment:
Use [minimist](https://github.com/substack/minimist) now that the args are a little more complex. This will reduce the amount of code to:

```js
var argv = require('minimist')(process.argv.slice(2), {boolean: 'dry-run'});
var dryRun = argv['dry-run'];
var cwd = argv._[0] || process.cwd();
```

Reply:
I was thinking the same thing about minimist, but that would add a dep for the CLI when typical usage is API. 

how about this?

```js
var argv = process.argv.slice(2);
var args = argv.filter(a => a !== '-d' && a !== '--dry-run');
var cwd = argv[0] || process.cwd();
var dryRun = argv.length !== args.length;
```

Reply:
I like it!